src/Map/src/Bridge/Google/README.md

```diff
@@ -10,19 +10,21 @@ UX_MAP_DSN=google://GOOGLE_MAPS_API_KEY@default
-| `nonce`    | Use a cryptographic nonce attribute                                                                                                |                                           |
-| `retries`  | The number of script load retries                                                                                                  | 3                                         |
-| `url`      | Custom url to load the Google Maps API script                                                                                      | `https://maps.googleapis.com/maps/api/js` |
-| `version`  | The release channels or version numbers                                                                                            | `weekly`                                  |
+| Option      | Description                                                                                                                        | Default                                                     |
+|-------------|------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------|
+| `id`        | The id of the script tag                                                                                                           | `__googleMapsScriptId`                                      |
+| `language`  | Force language, see [list of supported languages](https://developers.google.com/maps/faq#languagesupport) specified in the browser | The user's preferred language                               |
+| `region`    | Unicode region subtag identifiers compatible with [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1)                           |                                                             |
+| `nonce`     | Use a cryptographic nonce attribute                                                                                                |                                                             |
+| `retries`   | The number of script load retries                                                                                                  | 3                                                           |
+| `url`       | Custom url to load the Google Maps API script                                                                                      | `https://maps.googleapis.com/maps/api/js`                   |
+| `version`   | The release channels or version numbers                                                                                            | `weekly`                                                    |
+| `libraries` | The additional libraries to load, see [list of supported libraries](https://googlemaps.github.io/js-api-loader/types/Library.html) | `['maps', 'marker']`, those two libraries are always loaded |
```

Comment:
Should that be in controller.json or something like that ?

Reply:
What do you mean?